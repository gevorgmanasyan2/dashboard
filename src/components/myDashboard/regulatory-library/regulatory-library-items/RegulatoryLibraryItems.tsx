import React, { FC } from 'react'
import './regulatoryLibraryItems.scss'
import RegulatoryLibraryItem from './regulatory-library-item/RegulatoryLibraryItem'

interface RegulatoryLibraryItemsProps {
  waiverCategory?: object[]
}

const RegulatoryLibraryItems: FC<RegulatoryLibraryItemsProps> = ({ waiverCategory }) => {
  return (
    <div className="regulatory__library__items__container">
      {waiverCategory &&
        waiverCategory.map((item: any) => (
          <RegulatoryLibraryItem key={item.id} id={item.id} name={item.name} waiverLibraryItems={item.waiver_library_items}/>
        ))}
    </div>
  )
}

export default RegulatoryLibraryItems
