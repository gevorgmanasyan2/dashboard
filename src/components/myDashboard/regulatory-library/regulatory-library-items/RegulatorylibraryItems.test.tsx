/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import RegulatoryLibraryItems from './RegulatoryLibraryItems'

it('should render MyDashboard component', () => {
  const mockItem = {
    waiverCategory: [{ id: 1, name: 'test' }],
  }
  expect(shallow(<RegulatoryLibraryItems {...mockItem} />)).toMatchSnapshot()
})
