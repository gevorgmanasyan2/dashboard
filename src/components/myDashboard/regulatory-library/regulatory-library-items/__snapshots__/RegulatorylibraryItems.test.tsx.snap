// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MyDashboard component 1`] = `
<div
  className="regulatory__library__items__container"
>
  <RegulatoryLibraryItem
    id={1}
    key="1"
    name="test"
  />
</div>
`;
